frontend/public/components/replicaset.jsx

```diff
@@ -42,16 +42,17 @@ const Details = ({ obj: replicaSet }) => {
     'annotations',
     'deployment.kubernetes.io/revision',
   ]);
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text="Replica Set Details" />
+        <SectionHeading text={t('workload~Replica Set Details')} />
```

Comment:
We want to use sentence case for almost all headings and labels

```suggestion
        <SectionHeading text={t('workload~ReplicaSet details')} />
```